ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/platform/server/api/services/grpc/GrpcCommandSubmissionServiceSpec.scala

```diff
@@ -44,6 +45,7 @@ class GrpcCommandSubmissionServiceSpec
         currentLedgerTime = () => Instant.EPOCH,
         currentUtcTime = () => Instant.EPOCH,
         maxDeduplicationTime = () => Some(Duration.ZERO),
+        submissionIdGenerator = () => Ref.SubmissionId.assertFromString("submissionId"),
```

Comment:
Could you add a test that asserts that inner mockCommandSubmissionService gets a request augmented with a submissionId in case the caller fails to provide one?

Reply:
Good point. Done 591a98361e338101cd8c13d3c678fc180a568e3f